### ISSUE TITLE
When adding a new instance default value for audit and error forwarding is incositent

### ISSUE THREAD

**Issue by mauroservienti** [Particular/ServiceControl#999]
When adding a new ServiceControl instance the default value for `error forwarding` is `off`, and for `audit forwarding` is unset.

![image](https://user-images.githubusercontent.com/1325611/29510046-c14f8d88-865a-11e7-8dff-900eb1c5bdf8.png)

We should make a better choice. I believe that `error forwarding` is more important as a decision than `audit forwarding` so if users are forced to make a choice it should be the other way round.

Thoughts?

/cc @sergioc

**Comment by sergioc** [comment]
Alternatively use no default for either of those queues and force the user to make a choice for both. May not be worth to assume a default for a task that is performed so infrequently.

**Comment by mikeminutillo** [comment]
Error Forwarding should be off unless the customer knows what they are doing. ServiceControl tightly manages errors to ensure that they only have one outstanding retry at a time. It does this by consuming them from the error queue and preventing the rest of the system from accessing them (except to issue a retry). By forwarding errors to an external queue we open ourselves up to the possibility that errors get retried more than once. Arguably, the only thing anyone would want to do with messages in an error queue is delete (archive) them, or retry them. Off is the correct default here.

For auditing, this is not so simple. In order to install ServiceControl you must have had an audit queue already and you may have been using those messages for something. Audit Forwarding was originally introduced into SC to allow you "tap in" to the existing audit stream without breaking it. Now, it's possible that with the platform being a bigger part of our offering that people are _only_ configuring auditing in order to communicate with SC. In that case, I think this should default to Off as well. But that does leave open the possibility that customers who _were_ already using the messages in their audit queue, suddenly have that stream dammed by an SC instance.

**Comment by sergioc** [comment]
It's a bit strange that one option has a default value, while the other requires an explicit choice. I'd ask "why do I have to choose something here?" and might even trip on an error there because I missed the requirement to make a choice. If I understand @mikeminutillo's comment correctly, having an error queue default set to OFF helps ensure the user doesn't enable an option that may not be needed. However, it does not educate the user about why should it be OFF in most cases, while for auditing it does require the user to be educated. I may be nitpicking here, but this seems a bit clunky to me.

**Comment by mauroservienti** [comment]
I agree with @sergioc, I'd like to see links to doco pointing to what each choice means and its impact (especially in the long term).
